PHC-Android/PHC/src/main/java/phc/android/Networking/RequestManager.java

```diff
@@ -273,4 +275,79 @@ public void requestServices(final String userId,
         serviceRequest.setTag(sTAG);
         sRequestQueue.add(serviceRequest);
     }
+
+    /**
+     * Used to fetch all the services. need endpoint
+     *
+     *
+     */
+  public void requestGetApplied(final String qrCode,
+                                final String userId,
+                                final String authToken,
+                                Response.Listener<JSONObject> responseListener,
+                                Response.ErrorListener errorListener){
+
+      StringBuilder buildUrl = new StringBuilder(BASE_URL);
+      buildUrl.append("/api/v1/event_registrations/get_applied");
+      buildUrl.append("/");
+      buildUrl.append("?");
+      buildUrl.append("Number__c=");
+      buildUrl.append(qrCode);
+
+      JsonObjectRequest searchRequest = new JsonObjectRequest(buildUrl.toString(),
+              null,
+              responseListener,
+              errorListener){
+      @Override
+      public Map<String, String> getHeaders(){
+          HashMap<String, String> header = new HashMap<String, String>();
+          header.put("user_id", userId);
+//          header.put("auth_token", "vqWbG-dyt-cu9d9zqt1fXw");
+          header.put("auth_token", authToken);
+          header.put("Accept", "*/*");
+          return header;
+      }
+    };
+      searchRequest.setTag(sTAG);
+      sRequestQueue.add(searchRequest);
+  }
+
+    /**
+     *
+     * Used to create a new object with checkout information. Similar to requestUpdateService
+     */
+    public void requestUpdateFeedback(HashMap<String, Object> params,
+                                      final String userId,
+                                      final String authToken,
+                                      Response.Listener<JSONObject> responseListener,
+                                      Response.ErrorListener errorListener){
+
+        Log.d("---------------", "IN REQUEST UPDATE FEEDBACK");
+        JsonObjectRequest createRequest = new JsonObjectRequest(BASE_URL + "/api/v1/event_registrations/update_feedback",
```

Comment:
"/api/v1/event_registrations/update_feedback" should be saved as a variable (e.g. UPDATE_FEEDBACK_ENDPOINT)